aws_cloudtrail_rules/aws_console_login_without_mfa.py

```diff
@@ -1,10 +1,14 @@
 from panther import lookup_aws_account_name
 from panther_base_helpers import deep_get
+from panther_oss_helpers import check_new_user
 
 
 def rule(event):
     if event.get("eventName") != "ConsoleLogin":
         return False
+    # If Account is less than 3 days old do not alert
+    if check_new_user(lookup_aws_account_name(event.get('recipientAccountId'))) == "new_user":
```

Comment:
How is the account id (or name, if it's mapped in the helper) is getting _into_ the kv table?

Reply:
You are correct, `root` does not appear to have an `ACCOUNT_CREATED` event.  This being the case `root` should not appear in the KV store it the current implementation.  I'm going to look into a solve for this,  for simplicity sake I'm going to create a new PR for those changes